CodeEdit/Features/OpenQuickly/Views/OpenQuicklyView.swift

```diff
@@ -7,12 +7,6 @@
 
 import SwiftUI
 
-extension URL: Identifiable {
```

Comment:
Was there a reason for this removal? Seems like it was used in more than one location (and made the `WelcomeView` require some ugly types).

Reply:
I initially was trying to get around a SwiftLint warning, but I am realizing that this was helpful and probably don't need to get rid of it to fix the warning. I've found a way to do that so I'll put it back in.

Reply:
I added `@retroactive` for now. Lmk if you know of a better way to handle this.

Reply:
It looks like tests fail if I add this. Removing it and moving it to Utils/Extensions/URL.